src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -121,13 +121,13 @@
     <sys:String x:Key="Deploy_Start_Deploy">Avvia _distribuzione</sys:String>
     <sys:String x:Key="Deploy_Reset">_Ripristina tutte le impostazioni</sys:String>
     <sys:String x:Key="Deploy_View_Code">_Visualizza codice XML</sys:String>
-    <sys:String x:Key="Deploy_Generate_Command">_Generate deploy commands</sys:String>
+    <sys:String x:Key="Deploy_Generate_Command">_Genera comandi distribuzione</sys:String>
     <sys:String x:Key="Deploy_Apply_Preferences">_Applica le preferenze per le applicazioni di Office</sys:String>
     <sys:String x:Key="Deploy_Export_Config">_Esporta configurazione</sys:String>
     <sys:String x:Key="Deploy_Import_Config">_Importa configurazione</sys:String>
     <sys:String x:Key="Deploy_Import_Config_Web">Importa configurazione dal _web</sys:String>
-    <sys:String x:Key="Deploy_Check_Updates">Check Office _updates</sys:String>
-    <sys:String x:Key="Deploy_Repair_Office">Re_pair Office</sys:String>
+    <sys:String x:Key="Deploy_Check_Updates">Verifica a_ggiornamenti Office</sys:String>
```

Comment:
The hotkey `_g` is same as `_G` which is using in line 124 `_Genera`.

Reply:
@YerongAI 

What character hotkey is available using the string "Verifica aggiornamenti Office" ?

Reply:
The menu zone is from line 122 to line 130. I think use `_f` is okay. `Veri_f`.

And you don't need to mention me everytime. I said this before.

Reply:
Done. Thanks.